doc/nrf/releases/release-notes-latest.rst

```diff
@@ -197,34 +321,108 @@ The following list summarizes the most important changes inherited from upstream
   * Fixed a possible NULL pointer dereference when using any of the ``fs_`` functions.
     The functions will now return an error code in this case.
   * Fixed a garbage-collection issue in the NVS subsystem.
+  * Added the Kconfig option :option:`CONFIG_FS_FATFS_EXFAT` for enabling exFAT support.
+  * Added support for file open flags to fs and POSIX API.
+
+* Management:
+
+  * MCUmgr:
+
+    * Moved mcumgr into its own directory.
+    * Switched UDP port to use the kernel stack.
+    * Added missing socket close in error path for SMP.
+
+  * Added support for Open Supervised Device Protocol (OSDP) (see :option:`CONFIG_OSDP`).
+
+  * updatehub:
+
+    * Moved updatehub from :file:`lib` to :file:`subsys/mgmt`.
+    * Fixed out-of-bounds access and added a return value check for ``flash_img_init()``.
+    * Fixed a ``getaddrinfo`` resource leak.
+
+* Settings:
+
+  * Updated the implementation to return an error rather than faulting if there is an attempt to read a setting from a channel that does not support reading.
+  * Disallowed modifying the content of a static subtree name.
+
+* LVGL:
+
+  * Updated the library to the new major release v7.0.2.
+  * Aligned LVGL Kconfig constants with suggested defaults from upstream.
+
+* Tracing:
+
+  * Updated the API to check if the init function exists prior to calling it.
+
+* Logging:
+
+  * Fixed immediate logging with multiple backends.
+  * Switched logging thread to use the kernel stack.
+  * Allowed users to disable all shell backends at once using :option:`CONFIG_SHELL_LOG_BACKEND`.
+  * Added a logging backend for the Spinel protocol.
+  * Fixed timestamp calculation when using NEWLIB.
+
+* Shell:
+
+  * Switched to use the kernel stack.
+  * Fixed the select command.
+  * Fixed prompting dynamic commands.
+
+* libc:
+
+  * Simplified newlib malloc arena definition.
 
 * Devicetree:
 
   * Removed all nRF-specific aliases to particular hardware peripherals, because they are no longer needed now that nodes can be addressed by node labels.
     For example, you should now use ``DT_NODELABEL(i2c0)`` instead of ``DT_ALIAS(i2c_0)``.
+    If you use something similar to ``source $(SOC_DIR)/<path>``, change it to ``rsource <relative>/<path>`` or similar.
```

Comment:
Is this a continuation of the first item in this list, or should it be a separate item?

Reply:
should go after `* BOARD, SOC, DTS, and ARCH roots can now be specified in each module's :file:`zephyr/module.yml` file (see :ref:`modules_build_settings`).` Patch introduced by https://github.com/zephyrproject-rtos/zephyr/pull/26715 + https://github.com/zephyrproject-rtos/zephyr/pull/27431/files
I moved to `build system ` as more relevant.